addons/web/controllers/main.py

```diff
@@ -64,6 +64,38 @@
 
 COMMENT_PATTERN = r'Modified by [\s\w\-.]+ from [\s\w\-.]+'
 
+
+def none_values_filtered(func):
+    @functools.wraps(func)
+    def wrap(iterable):
+        return func(filter(lambda v: v is not None, iterable))
```

Comment:
We try to avoid using `filter` in our codebase
```suggestion
        return func(v for v in iterable if v is not None)
```